include/faker-cxx/types/Sex.h

```diff
@@ -8,13 +8,52 @@ enum class Sex
     Female,
 };
 
-inline std::string toString(Sex sex)
+const std::map<Language, std::map<Sex, std::string>> sexTranslations = {
+    {Language::English, {{Sex::Male, "Male"}, {Sex::Female, "Female"}}},
+    {Language::Polish, {{Sex::Male, "Mężczyzna"}, {Sex::Female, "Kobieta"}}},
+    {Language::Italian, {{Sex::Male, "Maschio"}, {Sex::Female, "Femmina"}}},
+    {Language::French, {{Sex::Male, "Homme"}, {Sex::Female, "Femme"}}},
+    {Language::German, {{Sex::Male, "Mann"}, {Sex::Female, "Frau"}}},
+    {Language::Russian, {{Sex::Male, "Мужчина"}, {Sex::Female, "Женщина"}}},
+    {Language::Romanian, {{Sex::Male, "Bărbat"}, {Sex::Female, "Femeie"}}},
+    {Language::Hindi, {{Sex::Male, "पुरुष"}, {Sex::Female, "महिला"}}},
+    {Language::Finnish, {{Sex::Male, "Mies"}, {Sex::Female, "Nainen"}}},
+    {Language::Nepali, {{Sex::Male, "पुरुष"}, {Sex::Female, "महिला"}}},
+    {Language::Spanish, {{Sex::Male, "Hombre"}, {Sex::Female, "Mujer"}}},
+    {Language::Turkish, {{Sex::Male, "Erkek"}, {Sex::Female, "Kadın"}}},
+    {Language::Czech, {{Sex::Male, "Muž"}, {Sex::Female, "Žena"}}},
+    {Language::Slovak, {{Sex::Male, "Muž"}, {Sex::Female, "Žena"}}},
+    {Language::Ukrainian, {{Sex::Male, "Чоловік"}, {Sex::Female, "Жінка"}}},
+    {Language::Danish, {{Sex::Male, "Mand"}, {Sex::Female, "Kvinde"}}},
+    {Language::Swedish, {{Sex::Male, "Man"}, {Sex::Female, "Kvinna"}}},
```

Comment:
Please restore Language enum as I removed it couple days ago, you can find it in github history it was commit like "remove language enum". Additionally please add more enum values based on Country enum as all countries should have corresponding language in Language enum.

Reply:
added Language.h and more enum values based on country enum